azurerm/resource_arm_container_group.go

```diff
@@ -80,6 +80,31 @@ func resourceArmContainerGroup() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"image_registry_credential": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"server": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
```

Comment:
Could we get some simple validation here (and on username/password)?

```go
ValidateFunc: validation.NoZeroValues
```

Would suffice